common/src/main/java/com/netflix/conductor/common/run/Workflow.java

```diff
@@ -319,8 +322,16 @@ public Set<String> getFailedReferenceTaskNames() {
         return failedReferenceTaskNames;
     }
 
-    public void setFailedReferenceTaskNames(Set<String> failedReferenceTaskNames) {
-        this.failedReferenceTaskNames = failedReferenceTaskNames;
+    public void setFailedReferenceTaskNames(Set<String> failedTaskNames) {
```

Comment:
nit: Is this variable name change really needed? I feel that the original retains readability.

Reply:
Good catch. That's a benign but unintended global search-n-replace error.